src/errorhandler/errorhandler.cpp

```diff
@@ -0,0 +1,87 @@
+#include "errorhandler.hpp"
+
+sErrorStructure cErrorHandler::ErrorStructure[E_NUMBER_OF_ERRORS] =
+{
+// Error number         |       Error active        |       Error enable        |       Reaction
+{  E_FREE_TO_USE,              false,                      E_ERROR_ENABLED,            NONE_REACT}
+};
+
+
+cErrorHandler::cErrorHandler(void) :
+    clearErrorTable(false)
+{
+    for(ui8 loopIndex = 0U; loopIndex < errorTableSize; loopIndex++)
+    {
+        errorTable[loopIndex] = E_NO_ERROR;
+    }
+}
+
+void cErrorHandler::SetErrorActive(eErrors arg_errorNumber)
+{
+    ErrorStructure[static_cast<ui8>(arg_errorNumber)].active = true;
```

Comment:
Use your already created abstraction: `SetError(arg_errorNumber, true);`. Same for `SetErrorDisactive`. If you will ever change your implementation you will do it on one place (`SetError`).